src/Core/src/Signer/SignerV4.php

```diff
@@ -77,9 +76,8 @@ public function sign(Request $request, Credentials $credentials, RequestContext
     {
         if (null === $now = $context->getCurrentDate()) {
             $now = new \DateTimeImmutable();
-        } else {
-            $now = new \DateTimeImmutable($now->format(\DateTimeInterface::ATOM));
         }
+
```

Comment:
```suggestion
        $now = $context->getCurrentDate() ?? new \DateTimeImmutable();
```

Reply:
I could not apply these with the GUI

<img width="383" alt="Screenshot 2020-03-27 at 00 47 45" src="https://user-images.githubusercontent.com/1275206/77707163-9c9c8d00-6fc4-11ea-942f-6f722207afb6.png">